### PR TITLE
spriggan and gown

### PR DESCRIPTION
**Pull request by melch-i**
added spriggan and gown to forest encounter pool.

### CODE REVIEW

File: classes/classes/Items/Armors/Gown.as [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+				"You dream about wandering around in the mountains. There is snow everywhere, and you notice something off in the distance digging its way out from under the snow. It’s a big bear and two smaller ones pawing their way out! You gleefully saunter up and say, 'Hello Mrs.bear, did you and your children enjoy your nap?'   You awake shortly after the strange dream, feeling confused. Was that really you?",
+				"You dream about sitting on a tree branch in your lovely gown. You freely hang off the large oak, dangling over the side while your legs hold you up.   Your gown falls down to your arms and hangs over your head. You get excited, pondering if someone might be watching and just got flashed.   You sit back up and straighten out your gown,  coy smile playing across your face.",
+			];
+			//display a random dream
+			clearOutput();
+			outputText("\n\n"+dryadDreams[rand(dryadDreams.length)]+"\n\n");
+			//build a list of non ideal parts 
+			if (player.hips.type != 5 )
```
Comment (Oxdeception) [inline classes/classes/Items/Armors/Gown.as]: Please use the constants here instead of bare numbers

File: classes/classes/Items/Armors/Gown.as [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+			return CoC.instance.player;
+		}
+		override public function useText():void{ //Produces any text seen when equipping the armor normally
+			switch (player.gender) {
+				case 2:   
+					outputText("You comfortably slide into the forest gown.   You spin around several times and giggle happily."
+					          +" Where did that come from?\n");
+					break;
+
+				case  1:
```
Comment (Oxdeception) [inline classes/classes/Items/Armors/Gown.as]: This should be using the gender constants

**Review by Oxdeception — CHANGES_REQUESTED** [review]
Please update code to use constants rather than straight ints.

**Review by Oxdeception — APPROVED** [review]

**Comment by melch-i** [comment]
I did change things and push to repo

**Pull request merged**